e2e/components/listing/details/criteria/criteria.po.js

```diff
@@ -0,0 +1,156 @@
+const elements = {
+  removedCriteria: '#removed-header',
+  attestToggle: '#success',
+  accept: '//span[text()="Accept"]',
+  testProcedureName: '#name',
+  version: '#version',
+  testToolsName: '#tt',
+  privacySecurityName: '#privacy-security-framework',
+};
+
+// Old elements added intentionally so it would be easy to clean up later
+const elementsOld = {
+  saveCertifiedProduct: 'button=Save Certification Criteria',
+  testProcedureName: '//*[@id="testProcedures "]',
+  testToolsName: '//*[@id="testTools "]',
+};
+
+class CriteriaComponent {
+  constructor() { }
+
+  get removedCriteriaHeader() {
+    return $(elements.removedCriteria);
+  }
+
+  editCriteria(id) {
+    $(`#criterion-id-${id}-edit`).scrollAndClick();
+  }
+
+  get attestToggle() {
+    return $(elements.attestToggle);
+  }
+
+  get accept() {
+    return $(elements.accept);
+  }
+
+  get saveCertifiedProductOld() {
+    return $(elementsOld.saveCertifiedProduct);
+  }
+
+  chipText(text) {
+    return $(`//span[text()="${text}"]`);
+  }
+
+  expandRemovedCriteria() {
+    $(elements.removedCriteria).$$('div')[1].scrollAndClick();
+  }
+
+  uiUpgradeFlag() {
+    if ($('chpl-certification-criteria').isExisting()) {
+      return false;
+    }
+    return true;
+  }
+
+  expandCriteria(id, criteria) {
+    if (this.uiUpgradeFlag()) {
+      $(`#criterion-id-${id}-header`).scrollIntoView();
+      $(`#criterion-id-${id}-header`).$$('div')[1].scrollAndClick();
+    } else {
+      $(`//*[@id="criteria_${criteria}_details_link"]`).scrollAndClick();
+    }
+  }
+
+  criteriaHeader(id, criteria, cures) {
+    if (this.uiUpgradeFlag()) {
+      return $(`#criterion-id-${id}-header`);
+    }
+    if (!this.uiUpgradeFlag()) {
+      if (cures) {
+        return $(`//*[@id="criteria_${criteria}_details_header_cures"]`);
+      }
+      return $(`//*[@id="criteria_${criteria}_details_header"]`);
+    }
```

Comment:
It's impossible for this first `if` to ever be false. If it were false, then we would already have returned from the if statement above

```suggestion
      if (cures) {
        return $(`//*[@id="criteria_${criteria}_details_header_cures"]`);
      }
      return $(`//*[@id="criteria_${criteria}_details_header"]`);
```